package.json

```diff
@@ -19,6 +19,7 @@
   },
   "homepage": "https://github.com/Skalar/eslint-config-skalar#readme",
   "dependencies": {
-    "babel-eslint": "^6.0.4"
+    "babel-eslint": "^6.0.4",
+    "eslint-plugin-babel": "^4.1.2"
```

Comment:
I am not sure it should be added as a dependency, since eslint-plugin-react is not added. But perhaps the only reason eslint-plugin-react is not added since many projects will not use it, in which case having this dependency explicitly included will make using this easier.